apps/dashboard/src/components/engine/badges/version.tsx

```diff
@@ -136,7 +141,12 @@ const UpdateVersionModal = (props: {
         dialogOverlayClassName="z-[10000]"
       >
         <DialogHeader>
-          <DialogTitle>Update Engine to {latest}?</DialogTitle>
+          <DialogTitle>Update Engine to {latestVersion}?</DialogTitle>
+
+          <DialogDescription>
+            {`It is recommended to pause traffic to Engine before performing this
+            upgrade. There is < 1 minute of expected downtime.`}
+          </DialogDescription>
```

Comment:
Don't need to wrap text with `{``}` here

Reply:
There's a `<` in the string, so I either need to do this or `&lt;`. I generally don't like the latter cause it makes it hard to grep for it in code.

Reply:
you should use the latter